src/components/navbar/Dropdown.scss

```diff
@@ -48,38 +50,31 @@
     right: 0;
     display: none;
     position: absolute;
-    background-color: var(--navbar-background-color);
-    border: 2px solid gray;
+    background-color: hsl(232.5, 9.8%, 16.1%);
```

Comment:
colors should be extracted somewhere right? but nice that you use hsl

Reply:
Done :)